### ISSUE TITLE
Node self connectinos

### ISSUE THREAD

**Issue by catena2w** [ScorexFoundation/Scorex#245]
I started 2 local nodes, `ergo-local-1` and `ergo-local-2`, but `/peers/connected` API route shows 3 active  connections:
```
[
  {
    "address": "",
    "lastSeen": 1526369972403,
    "name": "ergo-local-1"
  },
  {
    "address": "",
    "lastSeen": 1526369972403,
    "name": "ergo-local-1"
  },
  {
    "address": "",
    "lastSeen": 1526369972403,
    "name": "ergo-local-2"
  }
]
```

**Comment by catena2w** [comment]
In logs there are messages, that node really send connections to self:
```
17:47:44.859 INFO  [or.default-dispatcher-661] s.c.n.PeerConnectionHandler - Send message MessageSpec(22: RequestModifier) to bench1/88.198.13.202:37326
17:47:44.859 INFO  [or.default-dispatcher-661] s.c.n.PeerConnectionHandler - Received message MessageSpec(22: RequestModifier) from bench1/88.198.13.202:9003
17:47:44.860 INFO  [tor.default-dispatcher-10] s.c.n.PeerConnectionHandler - Send message MessageSpec(33: Modifier) to bench1/88.198.13.202:9003
17:47:44.860 INFO  [or.default-dispatcher-658] s.c.n.PeerConnectionHandler - Received message MessageSpec(33: Modifier) from bench1/88.198.13.202:37326
```

**Comment by catena2w** [comment]
relates to #296

**Comment by kushti** [comment]
will be finally fixed with #382
